app/javascript/packs/application.js

```diff
@@ -19,6 +19,8 @@ require('../partials/_nested_forms');
 require('../partials/_nested_content_block_form');
 require('../partials/_nested_lunch_form');
 
+require('../partials/_leaflet_map');
```

Comment:
is a map used already somewhere?

Reply:
Das war noch ein nicht gestashter Code Part für die spätere Leaflet Karte. Ich habe es rausgenommen.